### DISCUSSION TITLE
How can you bind to listen to a specific IP?

### DISCUSSION THREAD

**Discussion by ZaDarkSide** [uNetworking/uWebSockets.js#383] [Q&A] [answered]
Basically I see no option for selecting to listen on a specific IP and PORT, you can select only a specific PORT.
The issue I am facing is that I have a single VPS with 2 IPs, but I want to run 2 software on PORT 443, one of them is using uWebSocket.js
The other software has the option to listen on a specific IP and PORT, but uWebSocket.js bind by default to IPv6 instead of IPv4 and uses `:::*` for listening for incoming connections

**Comment by pavulon** [comment, marked as answer]
The docs are not 100% up to date, but it's there
https://unetworking.github.io/uWebSockets.js/generated/interfaces/templatedapp.html#listen

**Comment by ZaDarkSide** [comment]
Thank you very much, it's exactly what I was looking for.
